account_reversal/account_reversal.py

```diff
@@ -60,7 +60,9 @@ def _move_reversal(self, reversal_date,
         self.ensure_one()
         period_obj = self.env['account.period']
 
-        if not reversal_period_id:
+        if reversal_period_id:
+            reversal_period_id = period_obj.browse([reversal_period_id])[0]
+        else:
```

Comment:
I put it here because the browse for reversal_journal_id is already in this method
https://github.com/acsone/account-financial-tools/blob/8.0-fix-reversal-period-ape/account_reversal/account_reversal.py#L72